### PR TITLE
London gosh brca refactoring

### PR DESCRIPTION
**Pull request by FrancescoNea**
PR for refactoring of London Gosh provider Brca records extractor

### CODE REVIEW

**Review by ollietulloch — APPROVED** [review]
Thanks @FrancescoNea

File: lib/import/brca/core/brca_handler_mapping.rb [hunk 1]
```diff
@@ -20,7 +20,8 @@ class BrcaHandlerMapping
           'RJ7' => Import::Brca::Providers::StGeorge::StGeorgeHandler,
           'RPY' => Import::Brca::Providers::RoyalMarsden::RoyalMarsdenHandler,
           'R1K' => Import::Brca::Providers::LondonKgc::LondonKgcHandler,
-          'RQ3' => Import::Brca::Providers::Birmingham::BirminghamHandlerNewformat
+          'RQ3' => Import::Brca::Providers::Birmingham::BirminghamHandlerNewformat,
+          'RP4' => Import::Brca::Providers::LondonGosh::LondonGoshHandler,
```
Comment (ollietulloch) [inline lib/import/brca/core/brca_handler_mapping.rb:24]: @FrancescoNea Rubocop error introduced here:

Avoid comma after the last item of a hash. convention

**Review by ollietulloch — CHANGES_REQUESTED** [review]
Hi @FrancescoNea 

Please address rubocop error in `BrcaHandlerMapping`

File: lib/import/brca/core/brca_handler_mapping.rb (hunk 1 above)
Comment (FrancescoNea) [reply lib/import/brca/core/brca_handler_mapping.rb]: Thank you @ollietulloch , it should now be resolved

Comment (ollietulloch) [reply lib/import/brca/core/brca_handler_mapping.rb]: Great, thanks - resolved in 2df6277

**Review by ollietulloch — APPROVED** [review]

**Pull request merged**